lib/services/insights/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "azure-mgmt-insights",
```

Comment:
Amar will talk to you, this naming might not be accurate because you also have non management related client such as "hdinsights"